canonical-web-and-design/cn.ubuntu.com
# Update deprecated classes

## Done

Update deprecated classes
- Updated `p-heading--number` to `p-heading--[]`
- Updated `p-button-neutral` to `p-button`
- Removed  `p-link--external`

##QA

- Go to https://cn-ubuntu-com-633.demos.haus/
- Check all pages look good without the removed/updated classes
